.circleci/src/jobs/deploy_backend.yml

```diff
@@ -18,8 +18,17 @@ steps:
       fingerprints:
         - "a1:3f:a7:c3:ff:12:40:1d:85:de:a7:ab:12:3f:cc:05"
   - run:
-      name: "SFTP upload"
+      name: "SFTP upload packages"
```

Comment:
Same as above, no need for quotes 
and file should be named "deliver backend"
and "install" should be moved to the top